docs/delegate.md

```diff
@@ -1,14 +1,13 @@
 # Delegation
 
-For a more flexible configuration, you can have `server_name`
-resources (eg: `@user:example.com`) served by a different host and
-port (eg: `synapse.example.com:443`).
+Without configuring delegation, homeservers will expect the server
```

Comment:
Ooof this sentence is a bit tortuous. Maybe we can rephrase a bit based on the words in federate.md:

> By default, other homeservers will expect to be able to reach yours via your `server_name`, on port 8448.
For example, if you set your `server_name` to `example.com` (so that your user names look like
`@user:example.com`), other servers will try to connect to yours at `https://example.com:8448/`.

wdyt?

Reply:
Yeah I wasn't sure how to phrase that one. Yours is way better, thanks!